### PR TITLE
Simplify the attributes sent to the entity_add handler

### PR DESCRIPTION
**Pull request by jhrozek**
# Summary

I have a rather large branch that works towards not using database access from
the webhook handler. These two patches were part of that branch and can be submitted separately.

The aim is to decrease the number of attributes we send to the handler as well
as make them more generic.

Related: #4327

## Change Type

- [ ] Bug fix (resolves an issue without affecting existing features)
- [ ] Feature (adds new functionality without breaking changes)
- [ ] Breaking change (may impact existing functionalities or require documentation updates)
- [ ] Documentation (updates or additions to documentation)
- [x] Refactoring or test improvements (no bug fixes or new functionality)

# Testing

as part of a larger branch + unit tests

# Review Checklist:

- [x] Reviewed my own code for quality and clarity.
- [ ] Added comments to complex or tricky code sections.
- [ ] Updated any affected documentation.
- [x] Included tests that validate the fix or feature.
- [ ] Checked that related changes are merged.

### CODE REVIEW

**Comment by jhrozek** [comment]
@blkt would you mind reviewing this? You probably know the code better than I

File: internal/reconcilers/entity_add.go [hunk 1]
```diff
@@ -52,21 +51,17 @@ func (r *Reconciler) handleEntityAddEvent(msg *message.Message) error {
...
-	var repoName string
+	var entName string
 	var ok bool
-	if repoOwner, ok = event.Entity["repoOwner"].(string); !ok {
-		return errors.New("invalid repo owner")
-	}
-	if repoName, ok = event.Entity["repoName"].(string); !ok {
-		return errors.New("invalid repo name")
+	if entName, ok = event.Entity[properties.PropertyName].(string); !ok {
+		return fmt.Errorf("invalid or unset entity name: %s", entName)
```
Comment (JAORMX) [inline internal/reconcilers/entity_add.go:57]: Will this continue using the name? Or will this switch to using the upstream ID?

Comment (jhrozek) [reply internal/reconcilers/entity_add.go:57]: interesting idea, this code goes to repositories.CreateRepository which uses the name as a property for RetrieveAllProperties. I'm not sure if that and especially the underlying code supports retrieving a repo by name. I need to check that.

Comment (JAORMX) [reply internal/reconcilers/entity_add.go:57]: It's not a big deal what property is used, as long as we use properties and not tie something to names.

**Comment by jhrozek** [comment]
I will take the feedback into account, but I'm also going to close this PR and resend it in another PR that will actually also take this change into use.

**Pull request closed without merging**